tests/framework-tests/preferences/js/LetterSpaceEnactorTests.js

```diff
@@ -0,0 +1,100 @@
+        gradeNames: ["fluid.prefs.enactor.letterSpace"],
+        model: {
+            value: 1
+        },
+        fontSizeMap: fluid.tests.enactors.utils.fontSizeMap
+    });
+
+    fluid.defaults("fluid.tests.letterSpaceTests", {
+        gradeNames: ["fluid.test.testEnvironment"],
+        components: {
+            letterSpace: {
+                type: "fluid.tests.prefs.enactor.letterSpaceEnactor",
+                container: ".flc-letterSpace",
+                createOnEvent: "{letterSpaceTester}.events.onTestCaseStart"
+            },
+            letterSpaceTester: {
+                type: "fluid.tests.letterSpaceTester"
+            }
+        }
+    });
+
+    fluid.defaults("fluid.tests.letterSpaceTester", {
+        gradeNames: ["fluid.test.testCaseHolder"],
+        modules: [{
+            name: "fluid.prefs.enactor.letterSpace",
+            tests: [{
+                expect: 10,
+                name: "Set letter space",
+                sequence: [{
+                    listener: "jqUnit.assert",
+                    event: "{letterSpaceTests letterSpace}.events.onCreate",
+                    args: ["The letter space enactor was created"]
+                }, {
+                    func: "fluid.tests.letterSpaceTester.assertLetterSpace",
+                    args: ["{letterSpace}", {value: 1, unit: 0}]
+                }, {
+                    func: "{letterSpace}.applier.change",
+                    args: ["value", 2]
+                }, {
+                    changeEvent: "{letterSpace}.applier.modelChanged",
+                    spec: {path: "value", priority: "last:testing"},
+                    listener: "fluid.tests.letterSpaceTester.assertLetterSpace",
+                    args: ["{letterSpace}", {value: 2, unit: 1}]
+                }, {
+                    func: "{letterSpace}.applier.change",
+                    args: ["value", -0.5]
+                }, {
+                    changeEvent: "{letterSpace}.applier.modelChanged",
+                    spec: {path: "value", priority: "last:testing"},
+                    listener: "fluid.tests.letterSpaceTester.assertLetterSpace",
+                    args: ["{letterSpace}", {value: -0.5, unit: -1.5}]
+                }, {
+                    funcName: "fluid.tests.letterSpaceTester.reset"
+                }]
+            }]
+        }]
+    });
+
+    fluid.tests.letterSpaceTester.assertLetterSpace = function (that, expectedModel) {
+        var pxVal = expectedModel.unit * 16; // convert from em to px
+        var expectedLetterSpace = pxVal ? pxVal + "px" : "0";
+        jqUnit.assertDeepEq("The model should be set correctly", expectedModel, that.model);
+        jqUnit.assertEquals("The letter-spacing css style should be set to " + expectedLetterSpace, expectedLetterSpace, that.root.css("letter-spacing"));
+        jqUnit.assertEquals("The letter-spacing of the content is set to " + expectedLetterSpace, expectedLetterSpace, that.container.css("letter-spacing"));
```

Comment:
This assertion assumes the container doesn't have `letter-spacing` css being set, which is not always true.